### PR TITLE
feat: aws va quickstart

### PR DESCRIPTION
**Pull request by jcountsNR**
# Summary

New AWS Verified Access quickstart. Currently creating as a draft Pending release date.

## Pre merge checklist

- [x] Did you check you NRQL syntax? - Does it work?
- [x] Did you include a Data source and Documentation reference?
- [x] Are all documentation links publicly accessible?
- [x] Did you check your descriptive content for [voice and tone](https://docs.newrelic.com/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/)?
- [x] Did you check your descriptive content for spelling and grammar errors?
- [x] Did you review your content with a subject matter expert? (e.g. a Browser agent quickstart is reviewed with a member of the Browser Agent team)

### Dashboards

- [x] Does the PR contain a screenshot for each of your dashboards?
- [x] Do your screenshots show data?
- [x] Has the [sanitization script](https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#dashboards) been run on each dashboard?

### Alerts

- [x] Did you check that your alerts actually work?

### CODE REVIEW

**Review by MichelLosier — CHANGES_REQUESTED** [review]
Hi @jcountsNR we're in the process of deprecating install plans in favor of Data Sources. 

Could we get an aws verified access data source included in here?

More info:
https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#data-sources
https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#data-sources-1

**Comment by jcountsNR** [comment]
@MichelLosier done.

File: data-sources/amazon-verified-access/config.yml
Comment (MichelLosier) [inline data-sources/amazon-verified-access/config.yml:7]: Not required, but worth considering here adding the 'aws' categoryTerm

```yml
categoryTerms:
  - aws
```

This makes sure the data source shows up in marketplace under the AWS category

[Category term mappings](https://api.newrelic.com/graphiql?#query=%7B%0A%20%20actor%20%7B%0A%20%20%20%20nr1Catalog%20%7B%0A%20%20%20%20%20%20categories%20%7B%0A%20%20%20%20%20%20%20%20terms%0A%20%20%20%20%20%20%20%20displayName%0A%20%20%20%20%20%20%7D%0A%20%20%20%20%7D%0A%20%20%7D%0A%7D%0A)

**Review by MichelLosier — APPROVED** [review]
Looks good! A non-blocking consideration though for the data source

File: data-sources/amazon-verified-access/config.yml
Comment (MichelLosier) [reply data-sources/amazon-verified-access/config.yml:7]: Note also, we are waiting for the docs publication as well before we merge

**Pull request merged**
